src/components/subsidy-requests/SubsidyRequestsContext.jsx

```diff
@@ -0,0 +1,43 @@
+import { createContext, useMemo } from 'react';
+import {
+  useSubsidyRequestConfiguration,
+  useSubsidyRequestsOverview,
+} from './data/hooks';
+
+export const SubsidyRequestsContext = createContext();
+
+export const useSubsidyRequestsContext = (enterpriseId) => {
```

Comment:
If this just a hook now versus a component like before, should it be in `./data/hooks` versus a component file?

Reply:
will leave as is for now